mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageDAO.java

```diff
@@ -324,6 +324,7 @@ private Mono<byte[]> buildContentRetriever(FetchType fetchType, BlobId headerId,
         switch (fetchType) {
             case FULL:
                 return getFullContent(headerId, bodyId);
+            case ATTACHMENTS_METADATA:
```

Comment:
If I am correct, we will have a separate method (eg: `getContent(headerId...)`) for `ATTACHMENTS_METADATA`. Correct?

Reply:
Not here. There diff come from this: https://github.com/apache/james-project/pull/1366/files#diff-d217454c389000012a2f12f4f04e3a87d296176626e7c72ac6e9cf848515802aR56 where we load the attachments metadata